R/plot_functions.R

```diff
@@ -245,10 +245,10 @@ plot_krona<-function(physeq,output,variable, trim=F){
   # Abundance and taxonomic assignations for each OTU are fetched
   # and written to a file that would be processed by Krona.
   for( lvl in levels(df[,2])){
-    write.table(
-      unique(
-        df[ which( df[,2] == lvl), -2]
-        ),
+      tt = df[which(df[, 2] == lvl), -2]
+      tt2 = tt[which(tt[,1]!=0),]
+
+    write.table(tt2,
```

Comment:
```suggestion
    write.table(
      df[which(df[, 2] == lvl & df[,1] != 0), -2]
```

Reply:
Both statement can be combine and there will be no need for intermediate `tt` or `tt2` dataframes. If you're ok with that

Reply:
No problem